infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -118,9 +118,8 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
 
         // Store clientID as a channel attribute.
         ctx.channel().attr(clientIdAttrKey).set(clientId);
-        log.info("channelRead: Handshake validated by Server.");
-        log.debug("channelRead: Sending handshake response: Node Id: {}, Corfu client version:" +
-                " {}, Corfu server version: {}", this.nodeId, Long.toHexString(corfuClientVersion),
+        log.debug("channelRead: Sending handshake response: Node Id: {}, client version:" +
+                " {}, server version: {}", this.nodeId, Long.toHexString(corfuClientVersion),
```

Comment:
I thought it could be better to keep the into level verification log, which only happens upon handshake so it won't heavily increase the log?

Reply:
I think the debug line is needed for identifying where the handshake is coming from. The first line is just redundant is that case.